### ISSUE TITLE
Error Message when trying the example code

### ISSUE THREAD

**Issue by Rapid1898-code** [yeongbin-jo/python-chromedriver-autoinstaller#19]
When trying the example code i get this error message (on Windows10 with Python 3.9):
`[18304:13012:0420/181540.181:ERROR:device_event_log_impl.cc(214)] [18:15:40.181] Bluetooth: bluetooth_adapter_winrt.cc:1072 Getting Default Adapter failed.`

Example Code:
`from selenium import webdriver
import chromedriver_autoinstaller

chromedriver_autoinstaller.install()  # Check if the current version of chromedriver exists
                                      # and if it doesn't exist, download it automatically,
                                      # then add chromedriver to path

driver = webdriver.Chrome()
driver.get("http://www.python.org")
assert "Python" in driver.title`

**Comment by navdeepgaur** [comment]
This error is unrelated to this library. Some googling suggests that this is due to not having admin privileges for chrome to interact with bluetooth adapter.

For all purposes, this error should not affect what you are trying to do, unless you are specifically doing something with bluetooth.
Link here describes how you can suppress the message, or how you can elevate the access to work around this: https://stackoverflow.com/a/63270005/2879215

tldr:
suppress the warning: 
```python
options = webdriver.ChromeOptions() 
options.add_experimental_option("excludeSwitches", ["enable-logging"])
driver = webdriver.Chrome(options=options,
```
run as root user or admin privileges.

**Comment by Rapid1898-code** [comment]
Thanks for the response - i tried it again with only these 2 lines

```
import chromedriver_autoinstaller
chromedriver_autoinstaller.install()
```

# Check if the current version of chromedriver exists
# and if it doesn't exist, download it automatically, then add chromedriver to path

After execution there is no error - but there is also no chromdriver.exe in the folder where i started the program.

**Comment by navdeepgaur** [comment]
From what I have seen it does not add it to system path, but you can get the path returned by install() call.

Try:

```python
import chromedriver_autoinstaller
print(chromedriver_autoinstaller.install())
```
This way you'd know where its putting the downloaded chromedriver.
This also means that you can use the install call as
```python
driver = Chrome(chromedriver_autoinstaller.install())
```

P.S. I think it worked now vs earlier because you use a VPN? Probably a corporate VPN that requires a proxy to connect to the outside internet?

**Comment by Rapid1898-code** [comment]
Hello - with the print statement i showed me that the chromdriver was installed inside my venv-folder:
C:\Users\SomeUser\Documents\DEV\.venv\testing\lib\site-packages\chromedriver_autoinstaller\91\chromedriver.exe

Is there any way that the chromdriver.exe gets installed in the same folder where the program is running?

**Comment by Rapid1898-code** [comment]
With that it really work it seems - thank you very much!

```
from selenium import webdriver
import chromedriver_autoinstaller
from selenium.webdriver.chrome.options import Options
# import os,sys

chromedriver_autoinstaller.install()

options = Options()
# cd = '/chromedriver.exe'
# path = os.path.abspath (os.path.dirname (sys.argv[0]))
driver = webdriver.Chrome (chromedriver_autoinstaller.install(), options=options)
driver.get("http://www.python.org")
assert "Python" in driver.title
```

**Comment by navdeepgaur** [comment]
Hi @yeongbin-jo I think this issue can be closed, thanks for the great lib!
